test-harness/index.ts

```diff
@@ -83,6 +83,10 @@ describe('harness', () => {
         }
         if (parsed.expect) {
           expect(output.body).toStrictEqual(expected.body);
+        } else if (parsed.expectKeysOnly){
+          const jsonOutput = JSON.parse(output.body);
+          const jsonExpected = JSON.parse(expected.body);
+          expect(Object.keys(jsonOutput)).toStrictEqual(Object.keys(jsonExpected));
```

Comment:
interesting that this works with iterators

Reply:
it doesn't. This is meant only to work for json objects. Can probably update the readme to make that clear.

Reply:
[ac63648](https://github.com/stoplightio/prism/pull/2041/commits/ac6364871b702aaf0f11ddd3915f251629334804)